rubrik_cdm/cluster.py

```diff
@@ -606,3 +608,58 @@ def read_only_authorization(self, username, timeout=15):
 
         self.log("read_only_authorization: Granting read-only privilages to user '{}'.".format(username))
         return self.post("internal", "/authorization/role/read_only_admin", config, timeout)
+
+    def cluster_node_id(self):
+        """ Returns a list of node ids from all the nodes in the cluster.
```

Comment:
Can you make sure the docstring aligns properly? Example, https://github.com/rubrikinc/rubrik-sdk-for-python/blob/0e03d64c06bf3975b24829d6ed39709746cf6b00/rubrik_cdm/cluster.py#L64
```

Reply:
@king7997 Still need to update the docstring alignment on `cluster_node_id`